cmd/up/up.go

```diff
@@ -328,6 +329,13 @@ func (up *upContext) activate(isRetry, autoDeploy, build bool) error {
 		}
 	}
 
+	if _, err := registry.GetImageTagWithDigest(ctx, up.Dev.Image.Name); err != nil {
```

Comment:
how much time is the image analysis adding to `okteto up`, for the case where the image does exist?

Reply:
very little, it is just an http call if the image is in the okteto registry